src/composable/useSearchDeveloper.js

```diff
@@ -0,0 +1,31 @@
+import sortBy from "lodash/sortBy";
+import Searcher from "@/util/Searcher";
+import store from "../store/store";
+
+const developerSearcher = new Searcher().setField("name").setField("login");
+
+export function useSearchDeveloper() {
+  function searchPopular(searchTerm) {
+    let developers = store.state.Developers.developers;
+    developerSearcher.setData(developers);
+
+    if (!searchTerm) {
+      developers = developers.slice(0, 10);
+    }
+
+    return developerSearcher.findAll(searchTerm).slice(0, 10);
```

Comment:
You are still searching, even aftre there's no search term. What I would suggest is to just:

`if (searchTerm) { developers = developerSearcher.findAll(searchTerm); }`

And then return `developers.slice(0, 10)`. There's no really need to slice in 2 different places when it can be done in a single place. Let me know what you think about it.

Reply:
Fixed